db/migrate/20220412150442_add_days_to_accept_invitations_to_configuations_and_invitations.rb

```diff
@@ -0,0 +1,12 @@
+class AddDaysToAcceptInvitationsToConfiguationsAndInvitations < ActiveRecord::Migration[6.1]
+  def change
+    add_column :configurations, :number_of_days_to_accept_invitation, :integer, default: 3
+    add_column :invitations, :number_of_days_to_accept_invitation, :integer
```

Comment:
Pourquoi ne pas mettre une valeur par défaut pour l'invitation ? Cela permettrait en plus d'éviter de mettre l'itération ci-dessous il me semble

Reply:
J'avais commencé comme ça mais je veux que ça raise si on essaye d'envoyer une invitation sans avoir passé un nombre de jours (ce qui voudrait dire que  qq  chose  se  passe  mal), or si je mets une valeur par défaut ce sera pas le cas.